model-optimizer/extensions/middle/EltwiseInputReshape.py

```diff
@@ -106,8 +106,10 @@ def find_and_replace_pattern(self, graph: Graph):
                 # Insert Reshape layer between data node and consumer
                 for shape_key in mapping.keys():
                     shape = list(shape_key)
-                    reshape = Reshape(graph, attrs={'name': 'EltwiseReshapeNormalization'})
-                    reshape_dim = Const(graph, {'value': shape}).create_node_with_data()
+                    reshape_name = 'EltwiseReshapeNormalization'
```

Comment:
The name should be based on the input node `node` name

Reply:
Name updated